runtime/pack/dist/opt/termdebug/plugin/termdebug.vim

```diff
@@ -1224,33 +1224,25 @@ def DeleteCommands()
   delcommand Winbar
 
 
-  if !empty(saved_K_map) && saved_K_map.buffer
-    # pass
-  elseif !empty(saved_K_map) && !saved_K_map.buffer
-    nunmap K
+  if !empty(saved_K_map) && !saved_K_map.buffer
     mapset(saved_K_map)
   elseif empty(saved_K_map)
     silent! nunmap K
   endif
```

Comment:
this was a "comment" or remark, that is a reminder to following new coming coder to notice that difference.